required/tools/longtable.dtx

```diff
@@ -907,6 +930,15 @@
 %    \end{macrocode}
 % \end{macro}
 %
+% \begin{macro}{\LTcaptype}
+% The name used as counter, in caption, "\addcontentsline" and in targets.
+% "\providecommand" is used for compability with \pkg{ltcaption}
```

Comment:
We typically use "|"

Reply:
@josephwright  not my fault ;-)

Reply:
longtable predates latex2e:-)  it has `% \DeleteShortVerb{\|}`  (it has several fragments that want a verbatim `|`)